docs/design/software-design-specification.adoc

```diff
@@ -83,7 +83,7 @@ Input parameters:
 | `build-go.sh` shell script
 a| Runs `gofmt` (SDS-EXT-3) to check all Go files are formatted.
 
-Runs `golangci-lint` (SDS-EXT-4) to check if there are any lint errors.
+Runs `golangci-lint` (SDS-EXT-4) to check if there are any lint errors. A report is placed into `.ods/artifacts/lint-reports`.
```

Comment:
Is that change so small that there is no need to update the version of the document to 0.10? If so, what is the threshold?

Reply:
There is no threshold. Every change should result in a history entry. Addressed in c7aa5a0.